doc/explanation/scopes.rst

```diff
@@ -0,0 +1,39 @@
+Dune Projects and Workspaces
+============================
+
+Whenever Dune builds anything, it does so at the level of a *Dune workspace*. A
+Dune workspace is a set of Dune projects. A typical workspace consists of a
+single Dune project.
+
+A Dune project is defined by the presence of a
```

Comment:
Just as we used italics for Dune workspace, we should use italics for Dune project since we are defining the term.

Reply:
Fixed, thanks.